app/views/questions/show.html.slim

```diff
@@ -33,7 +33,7 @@
         | )
       = render partial: 'answers/answer', collection: @question.answers
 
-    - if user_signed_in?
+    - if user_signed_in? && !@question.expired?
```

Comment:
I'm not sure what we can check in the controller action or what we would do with that information. Could you elaborate? 

I  did add a new validation on answers to make sure users cant answer closed questions.